debian/mce.init

```diff
@@ -1,4 +1,14 @@
 #!/sbin/openrc-run
+### BEGIN INIT INFO
+# Provides:          mce
+# Required-Start:    $remote_fs dsme dbus dbus-user
+# Required-Stop:     $remote_fs dsme dbus dbus-user
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Mode Control Entity
+# Description:       This init script starts the mode control software
+#                    used on the Maemo platform for Internet Tablets
+### END INIT INFO
```

Comment:
So I did this for one or two packages because @parazyd asked me to do it, but it seems to break even just running the scripts with openrc. I am not sure if you tested this, or are just adding it because I added to a few packages, but probably don't want to spend a lot more time doing this right now.

Reply:
I do this because missing LSB headers breaks systemd setup as well. It doesn't allow to enable/disable service because of errors like this:
update-rc.d: error: mce Default-Start contains no runlevels, aborting.

Reply:
> So I did this for one or two packages because @parazyd asked me to do it, but it seems to break even just running the scripts with openrc. I am not sure if you tested this, or are just adding it because I added to a few packages, but probably don't want to spend a lot more time doing this right now.

How to reproduce the errors you get ("it seems to break even just running the scripts with openrc")?

Reply:
Let's move this discussion to an issue/mail/irc, since we'll definitely want to record this somewhere, and also get Parazyd's input. I need to go out for ~1 hour or so, but will be around after that.

In my case, running `/etc/init.d/droid4-powermanagement status` didn't work when the lsb headers were present. Removing them made it work. Perhaps `runscript` should be used rather than `openrc-run`.

Reply:
I am OK with the PR, apart from these LSB headers. This currently will break builds. So please remove it for now, and let's add it later when we're all ready for this.

Reply:
> I am OK with the PR, apart from these LSB headers. This currently will break builds. So please remove it for now, and let's add it later when we're all ready for this.

Done.

Reply:
BTW, I already debugged openrc and know how to fix LSB headers not working with it.
All you need is just comment or remove 321 - 322 lines from /lib/rc/sh/openrc-run.sh

Reply:
cc @parazyd ^^